pallets/runtime/common/src/runtime.rs

```diff
@@ -177,18 +177,20 @@ macro_rules! misc_pallet_impls {
             type WeightToFeeConst = polymesh_runtime_common::WeightToFee;
         }
 
-        impl pallet_balances::BlockRewardConfig for Runtime {
-            type BlockRewardsReserve = pallet_balances::Pallet<Runtime>;
-        }
-
         impl pallet_balances::Config for Runtime {
-            type MaxLocks = MaxLocks;
+            type Balance = Balance;
             type DustRemoval = ();
             type RuntimeEvent = RuntimeEvent;
+            #[cfg(not(feature = "runtime-benchmarks"))]
             type ExistentialDeposit = ExistentialDeposit;
+            #[cfg(feature = "runtime-benchmarks")]
```

Comment:
Avoids having to change multiple benchmarks in the substrate version. This is only used to set an initial balance in the benchmarks so there's no problem having a value greater than zero here.